package.json

```diff
@@ -1,110 +1,138 @@
   },
   "devDependencies": {
     "@action-validator/cli": "^0.5.3",
     "@action-validator/core": "^0.5.3",
-    "@babel/core": "^7.22.10",
-    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
-    "@storybook/addon-actions": "^6.5.10",
-    "@storybook/addon-essentials": "^6.5.10",
-    "@storybook/addon-knobs": "^7.0.0",
-    "@storybook/addon-links": "^6.5.10",
-    "@storybook/builder-webpack5": "^7.3.1",
-    "@storybook/manager-webpack5": "^6.5.10",
-    "@storybook/preset-create-react-app": "4.1.2",
-    "@storybook/react": "^6.5.10",
-    "@testing-library/dom": "^9.3.1",
-    "@testing-library/jest-dom": "^5.16.4",
-    "@testing-library/react": "^11.2.7",
+    "@babel/core": "^7.14.5",
+    "@babel/preset-react": "^7.14.5",
+    "@commitlint/cli": "^17.0.0",
+    "@commitlint/config-conventional": "^17.0.0",
+    "@jscutlery/semver": "^3.1.0",
+    "@nx/eslint-plugin": "16.8.0",
+    "@nx/js": "16.8.0",
+    "@nx/linter": "16.8.0",
+    "@nx/react": "16.8.0",
+    "@nx/storybook": "16.8.0",
+    "@nx/vite": "16.8.0",
+    "@nx/workspace": "16.8.0",
+    "@swc/cli": "~0.1.62",
+    "@swc/core": "~1.3.51",
+    "@testing-library/jest-dom": "^6.1.3",
+    "@testing-library/react": "14.0.0",
     "@testing-library/user-event": "^14.4.3",
-    "@types/jest": "^29.5.3",
+    "@types/classnames": "^2.3.1",
     "@types/node": "^20.5.3",
     "@types/react": "^18.2.21",
     "@types/react-dom": "^18.2.7",
-    "@types/react-router": "^5.1.20",
-    "@types/react-router-dom": "^5.3.3",
-    "babel-loader": "^9.1.3",
+    "@typescript-eslint/eslint-plugin": "^5.60.1",
+    "@typescript-eslint/parser": "^5.60.1",
+    "@vitejs/plugin-react": "~4.0.0",
+    "@vitest/coverage-c8": "~0.32.0",
+    "@vitest/ui": "~0.32.0",
     "chokidar-cli": "^3.0.0",
     "colors": "^1.4.0",
     "concurrently": "^8.2.1",
     "copyfiles": "^2.4.1",
     "cross-env": "^7.0.3",
     "documentalist": "^1.6.0",
-    "eslint-plugin-babel": "^5.3.1",
+    "eslint": "~8.46.0",
+    "eslint-config-prettier": "8.1.0",
+    "eslint-plugin-import": "2.27.5",
+    "eslint-plugin-jsx-a11y": "6.7.1",
+    "eslint-plugin-playwright": "^0.15.3",
+    "eslint-plugin-react": "7.32.2",
+    "eslint-plugin-react-hooks": "4.6.0",
     "generate-changelog": "^1.8.0",
     "husky": "^8.0.3",
-    "lerna": "^6.6.1",
+    "hyperform": "^0.12.1",
+    "jsdom": "~22.1.0",
     "module-alias": "^2.2.2",
-    "prettier": "2.3.1",
-    "react": "^18.2.0",
-    "react-app-rewired": "^2.2.1",
-    "react-dom": "^18.2.0",
-    "react-ga": "^3.3.0",
-    "react-live": "^3.1.0",
-    "react-markdown": "^6.0.2",
-    "react-router-dom": "^6.3.0",
-    "react-scripts": "^5.0.1",
+    "ngx-deploy-npm": "^7.0.1",
+    "nx": "16.8.0",
+    "prettier": "^2.6.2",
     "sass": "^1.54.4",
```

Comment:
This is the dependency that is currently causing breaking changes, as we now have to install sass in order to use v14. Can we somehow check whether that is necessary anymore?

Reply:
Using Vite might fix this, we can verify after releasing a new alpha version